modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc

```diff
@@ -15,6 +15,17 @@ You can install {prod-short} on OpenShift.
 
 * `{prod-cli}`. See: xref:installing-the-chectl-management-tool.adoc[].
 
+[IMPORTANT]
+====
+Ensure that there are no `CheCluster` API by running the following command:
+[source,subs="+quotes,attributes"]
+----
+$ {orch-cli} get crd checlusters.org.eclipse.che
+Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "checlusters.org.eclipse.che" not found
```

Comment:
Does running `dsc` result in exactly the same message? In particular the `"checlusters.org.eclipse.che"` part?

Reply:
`orch-cli` stands for `kubectl/oc`. The output is the same for both tools.